src/main/java/MediumDificultyAssignment/PrimeNumber.java

```diff
@@ -4,23 +4,26 @@
 
 public class PrimeNumber {
     public void calculatePrimeNum(int[] inputArray) {
+        int count = 0, varOne = 1;
         int lastNum = inputArray[inputArray.length - 1];
-        int resultNum = lastNum <= 1 ? 1 : lastNum == 2 ? 2 : 3;
+        int resultNum = lastNum <= 1 ? 1 : 2;
         switch (resultNum) {
             case 1:
-                System.out.println("The last number of array is not prime number: " + lastNum);
+                System.out.println(lastNum + " is a Not a prime number ");
                 break;
             case 2:
-                System.out.println("The last number of array is prime number: " + lastNum);
-                break;
-            case 3:
-                if ((lastNum / 2) % 2 == 0) {
-                    System.out.println("The last number of array is not prime number: " + lastNum);
-                } else {
+                while (varOne <= lastNum) {
+                    if (lastNum % varOne == 0) {
+                        count++;
+                    }
```

Comment:
Please do not use loop to find prime number

Reply:
code updated.